docs/manuals/user/index.md

```diff
@@ -141,29 +141,46 @@ You need to make sure your NGSI entities are using the valid NGSI types for the
-|Number              | [float](https://crate.io/docs/crate/reference/sql/data_types.html#numeric-types)                   |-|
-|Text                | [string](https://crate.io/docs/crate/reference/sql/data_types.html#string)                  | This is the default type if the provided NGSI Type is unsupported or wrong. |
+|Number              | [real](https://crate.io/docs/crate/reference/sql/data_types.html#numeric-types)                   |-|
+|Text                | [text](https://crate.io/docs/crate/reference/sql/data_types.html#data-type-text)                  | This is the default type if the provided NGSI Type is unsupported or wrong. |
 |StructuredValue     | [object](https://crate.io/docs/crate/reference/sql/data_types.html#object)                  |-|
 
+**TimescaleDB (v12.x) Translation Table**
+
+| NGSI Type          | TimescaleDB Type          | Observation |
+| ------------------ |:-----------------------:| :-----------|
+|Array               | [jsonb](https://www.postgresql.org/docs/current/datatype-json.html)           |  |
+|Boolean             | [boolean](https://www.postgresql.org/docs/current/datatype-boolean.html)                 | - |
+|DateTime            | [timestamp WITH TIME ZONE](https://www.postgresql.org/docs/current/datatype-datetime.html)                 | 'ISO8601' can be used as equivalent of 'DateTime'. |
+|Integer             | [bigint](https://www.postgresql.org/docs/current/datatype-numeric.html#DATATYPE-INT)                  | - |
+|[geo:point](http://docs.orioncontextbroker.apiary.io/#introduction/specification/geospatial-properties-of-entities)            | [geometry](https://postgis.net/docs/geometry.html)               | **Attention!** NGSI uses "lat, long" order whereas PostGIS/WGS84 stores points in [long, lat] order.|
+|[geo:json](http://docs.orioncontextbroker.apiary.io/#introduction/specification/geospatial-properties-of-entities)            | [geometry](https://postgis.net/docs/geometry.html)               | - |
+|Number              | [float](https://www.postgresql.org/docs/current/datatype-numeric.html#DATATYPE-FLOAT)                   |-|
+|Text                | [text](https://www.postgresql.org/docs/current/datatype-character.html)                  | This is the default type if the provided NGSI Type is unsupported or wrong. |
+|StructuredValue     | [jsonb](https://www.postgresql.org/docs/current/datatype-json.html)                  |-|
+
+
+
 If the type of any of the received attributes is not present in the column
 *NGSI Type* of the previous table, the value of such attribute will be treated
 internally as a string. So, if you use `Float` for your attribute type (not
-valid), your attribute will be stored as a `string`.
+valid), your attribute will be stored as a `text`.
```

Comment:
if type not a NGSI type, the is inferred. with few exceptions to provide NGSI-LD forward compatibility